<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COPSTACDialog</class>
 <widget class="QDialog" name="COPSTACDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to STAC COP Format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupLayers">
     <property name="title">
      <string>Layer Selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listLayers"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnSelectAll">
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeselectAll">
          <property name="text">
           <string>Deselect All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCOP">
     <property name="title">
      <string>COP Metadata</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMission">
        <property name="text">
         <string>Mission:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineMission">
        <property name="placeholderText">
         <string>Mission or operation identifier</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelClassification">
        <property name="text">
         <string>Classification:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboClassification">
        <item>
         <property name="text">
          <string>public release</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>internal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>confidential</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>restricted</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>classified</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelReleasability">
        <property name="text">
         <string>Releasability:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineReleasability">
        <property name="placeholderText">
         <string>e.g., 1:N or specific entity IDs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDGGSCRS">
        <property name="text">
         <string>DGGS CRS:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineDGGSCRS">
        <property name="placeholderText">
         <string>e.g., rHEALPix-R12</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDGGSZone">
        <property name="text">
         <string>DGGS Zone ID:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineDGGSZone">
        <property name="placeholderText">
         <string>DGGS zone identifier (optional)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelServiceProvider">
        <property name="text">
         <string>Service Provider:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineServiceProvider">
        <property name="placeholderText">
         <string>Data/service provider (optional)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOutput">
     <property name="title">
      <string>Output Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineOutputDir">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Select output directory...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectOutput">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCreateZip">
        <property name="text">
         <string>Create ZIP archive of exported files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
